bin/rmt-mirror

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env ruby
 
-require_relative '../config/boot'
-$LOAD_PATH.unshift File.expand_path(File.join(__dir__, '../lib/'))
+require_relative '../config/environment'
```

Comment:
I like it, but `time bin/rmt-repos` went from 0.8 sec to 1.5 sec (which doesn't do anything and just prints help message), which is somewhat annoying. :-/
We don't need to load all of Rails dependencies, just config and ActiveRecord.